pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,569 @@
+#!/bin/bash
+# Created by Roel Van de Paar, Percona LLC
+
+# This script quickly and intelligently generates all available mysqld --option combinations (i.e. including values)
+
+# User variables
+OUTPUT_FILE=/tmp/mysqld_options.txt
```

Comment:
Retain the same name: mysqld_options_ms_80.txt

Reply:
Fixed